src/generated/java/org/lwjglx/opengl/GL20.java

```diff
@@ -154,6 +154,13 @@ public static void glGetActiveUniform(int program, int index, java.nio.IntBuffer
         org.lwjgl.opengl.GL20.glGetActiveUniform(program, index, length, size, type, name);
     }
 
+    public static String glGetActiveUniform(int program, int index, int maxLength, java.nio.IntBuffer sizeType) {
+        java.nio.IntBuffer type = org.lwjgl.BufferUtils.createIntBuffer(1);
```

Comment:
Worth reusing a static buffer for this? :thinking:

Reply:
possibly, a ThreadLocal would make sense